client/src/assets/i18n/en.json

```
@@ -572,6 +572,7 @@
   "fight.step.stun": "{{brute}} was stunned.",
   "fight.step.haste": "{{brute}} rushed past {{opponent}} and dealt {{damage}} damage.",
   "fight.step.treat": "{{brute}} gave their pet {{target}} a treat and healed them for {{heal}}HP.",
+  "fight.step.dropShield": "{{brute}} dropped it's shield.",
```

Using neutral pronouns is a better choice whenever possible

```suggestion
  "fight.step.dropShield": "{{brute}} dropped their shield.",
```